grow/translators/google_sheets.py

```diff
@@ -351,9 +355,30 @@ def _generate_style_requests(self, sheet_id):
             },
         }
 
+        formats['missing_cell'] = {
+            'backgroundColor': {
+                'red': 1,
+                'green': 0.95,
+                'blue': 1
+            }
+        }
+
         formats['wrap'] = {'wrapStrategy': 'WRAP'}
 
         requests = []
+
+        # TODO Figure out how to be smarter about matching conditional formatting.
```

Comment:
True, it just bugs me because if anyone wants to do something custom to their sheet we wipe it out. Not that it is always bad, just annoying to me that it can't be more precise :)